src/rapaio/math/linear/dense/LUDecompositionData.java

```diff
@@ -0,0 +1,54 @@
+package rapaio.math.linear.dense;
```

Comment:
Similar with a previous comment. What it is the advantage of adding another data structure in LU decomposition? All calls will be indirected through this class and the size of the original code will remain the same. I think is better to keep it like it was, for speed if nothing else.